ted_sws/data_manager/services/create_notice_collection_materialised_view.py

```diff
@@ -33,6 +31,23 @@ def create_notice_collection_materialised_view(mongo_client: MongoClient):
                 "publication_date": "$normalised_metadata.publication_date",
             }
         },
+        {
+            "$out": NOTICES_MATERIALISED_VIEW_NAME
+        }
+    ])
+    materialised_view = database[NOTICES_MATERIALISED_VIEW_NAME]
+    materialised_view.create_index([("created_at", DESCENDING)])
+    materialised_view.create_index([("publication_date", DESCENDING)])
+    materialised_view.create_index([("eu_institution", ASCENDING)])
+    materialised_view.create_index([("status", ASCENDING)])
+    materialised_view.create_index([("form_number", ASCENDING)])
+    materialised_view.create_index([("form_number", ASCENDING), ("status", ASCENDING)])
+
+
+def update_notice_collection_materialised_view(mongo_client: MongoClient):
+    database = mongo_client[config.MONGO_DB_AGGREGATES_DATABASE_NAME or "aggregates_db"]
```

Comment:
don't use default strings like ("aggregates_db") ase database name, in anycase use constant var like as DEFAULT_AGGREGATES_DB_NAME instead of "aggregates_db".

Reply:
done